tools/Az/Az.psd1

```diff
@@ -53,18 +53,22 @@ DotNetFrameworkVersion = '4.7.2'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.0.0'; },
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.0.1'; },
```

Comment:
do we have az.resources module in this list?

Reply:
yes we have - we are using version 0.11.0 for Az.Resources